jablib/README.md

```diff
@@ -1,5 +1,11 @@
 # `jablib`
 
+`jablib` offers JabRef's model and logic code as a Java library.
+All code in `src/main/java` is published as a Maven artifact at `org.jabref:jablib`.
+Example code is made available at <https://github.com/JabRef/jabref/tree/main/jablib-examples>.
+
+You can browse JavaDoc at APIdia: <https://apidia.net/java/JabRef/6.0-snapshot-2025-12-02/?cls=org.jabref.jablib-module>.
```

Comment:
Generally, whenever you use a version on APIdia urls, you can and should omit the lower part - as much of it as possible. For underspecified versions, APIdia's server defaults to the newest lower part. In this case, you'd want plainly `6` instead of `6.0-snapshot-2025-12-02`. You can use `main` to even keep it pointing to newest snapshots one day when `7` is the leading major version. So after the latest enhancement, this link can be `https://apidia.net/java/JabRef/jablib/main`. (Plainly `https://apidia.net/java/JabRef/jablib` would be similar to `https://apidia.net/java/JabRef/jablib/latest`, which would keep pointing to `6` once a `7.0-snapshot` train starts, i.e. to the then-latest non-snapshot release of jablib.)